src/jit/lsra.cpp

```diff
@@ -1868,6 +1868,14 @@ void LinearScan::identifyCandidates()
             continue;
         }
 
+#if FEATURE_MULTIREG_ARGS_OR_RET        
+        if (varDsc->lvIsMultiRegArgOrRet)
+        {
+            varDsc->lvLRACandidate = 0;
+            continue;
```

Comment:
As per my understanding, this too is a short-term change and should be removed in near future?  In fact in importer we already set DoNotEnregister in most cases.  Having this change LSRA would mean that now we have this short-term code spread around in more places than we like.